src/gl/rs-gl.cpp

```diff
@@ -114,7 +114,7 @@ int rs2_gl_is_frame_extendable_to(const rs2_frame* f, rs2_gl_extension extension
         if (!gpu->get_gpu_section().on_gpu()) return false;
         return true;
     }
-    default: return 0;
+    default: return false;
```

Comment:
@battlecry231 , this seem little weird as one would except that the compiler complains about the `return false` in the above `case RS2_GL_EXTENSION_VIDEO_FRAME` statement.
The default return (int) is actually consistent with the function's signature.
Can you post the Make error you see ?

Reply:
Here is the error when I try to compile withe `default: return 0;`:
```
[ 54%] Building CXX object src/gl/CMakeFiles/realsense2-gl.dir/rs-gl.cpp.o
/home/***/Downloads/librealsense/src/gl/rs-gl.cpp: In lambda function:
/home/***/Downloads/librealsense/src/gl/rs-gl.cpp:117:21: error: inconsistent types ‘bool’ and ‘int’ deduced for lambda return type
     default: return 0;
                     ^
make[2]: *** [src/gl/CMakeFiles/realsense2-gl.dir/build.make:102: src/gl/CMakeFiles/realsense2-gl.dir/rs-gl.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:407: src/gl/CMakeFiles/realsense2-gl.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```